src/main/java/com/mindee/utils/DeserializationUtils.java

```diff
@@ -12,25 +12,31 @@
 import com.mindee.model.fields.PaymentDetails;
 import com.mindee.model.fields.Tax;
 import com.mindee.model.fields.Time;
+import com.mindee.model.geometry.Polygon;
 import com.mindee.model.ocr.PageContent;
+import com.mindee.utils.geometry.PolygonUtils;
 import java.io.IOException;
 import java.time.LocalDate;
 import java.time.LocalTime;
 import java.time.format.DateTimeFormatter;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import lombok.Singular;
+import lombok.SneakyThrows;
 
 public class DeserializationUtils {
 
   private static ObjectMapper mapper = new ObjectMapper();
 
-  public static Amount amountFromJsonNode(JsonNode amountPrediction) throws IOException {
+  @SneakyThrows
```

Comment:
What is the meaning of this annotation ?

Reply:
sneakythrows is useful to wrap checked exceptions into runtime exceptions in scenarios like these were the caller can't really handle these and in scenarios like this where exception is only theoretically possible, but not practically possible